platform/viewer/src/components/ViewportGrid.jsx

```diff
@@ -194,14 +194,15 @@ function ViewerViewportGrid(props) {
           isActive={isActive}
         >
           <div
-            className={classNames('h-full w-full flex flex-col align-center', {
+            className={classNames('h-full w-full flex flex-col', {
               'pointer-events-none': !isActive,
             })}
           >
             <ViewportComponent
               displaySet={displaySet}
               viewportIndex={viewportIndex}
               dataSource={dataSource}
+              commandsManager={commandsManager}
```

Comment:
I'm pretty sure the `ViewportCompnent` is the component exported/registered by the extension. Each extension module receives the `commandsManager` and `servicesManager`. If we need to pass them, we should do it _in the extension_ instead of making it an app concern that impacts _all_ ViewportComponents, as they may not all use/need them.

Reply:
Updated, thanks.